src/storage/src/hummock/compactor/mod.rs

```diff
@@ -249,6 +244,10 @@ impl Compactor {
                 return task_status;
             }
         }
+
+        let (input_memory, total_file_count, total_key_count) =
```

Comment:
I think this limit is a bit too aggressive, for each merge_iter , each sst will only hold the memory of one block. Do we need to discuss how to evaluate the memory of a task? @hzxa21 @wenym1

Reply:
+1. I think we need to do experiment on the peak memory usage of a task with N SSTs in overlapping levels and M SSTs in non-overlapping levels.